postgres/osfamilymap.yaml

```diff
@@ -21,10 +21,17 @@ Debian:
     {% endif %}
 
 FreeBSD:
-  conf_dir: /usr/local/pgsql/data
-  data_dir: /usr/local/pgsql/data
-  user: &freebsd-user pgsql
-  group: &freebsd-group pgsql
+    {% if repo.version >= 9.6 %}
+  user: &freebsd-user postgres
+  group: &freebsd-group postgres
+  conf_dir: {{ '/var/db/postgres/data' ~ release  }}
+  data_dir: {{ '/var/db/postgres/data' ~ release }}
+    {% else %}
+  user: &freebsd-user {{ pgsql }}
+  group: &freebsd-group {{ pgsql }}
+  conf_dir: {{ '/usr/local/pgsql/data' }}
+  data_dir: {{ '/usr/local/pgsql/data' }}
```

Comment:
why use string interpolation for static strings?

Reply:
No reasons, I  modified

Reply:
thank you!